sycl/include/CL/sycl/handler.hpp

```diff
@@ -959,7 +961,7 @@ class __SYCL_EXPORT handler {
             int Dims, typename Reduction>
   detail::enable_if_t<Reduction::accessor_mode == access::mode::read_write &&
                       Reduction::has_fast_atomics && !Reduction::is_usm>
-  parallel_for(nd_range<Dims> Range, Reduction Redu, KernelType KernelFunc) {
+  parallel_for(nd_range<Dims> Range, Reduction Redu, const KernelType KernelFunc) {
```

Comment:
```suggestion
  parallel_for(nd_range<Dims> Range, Reduction Redu, const KernelType &KernelFunc) {
```
The SYCL 2020 provisional says it should be a reference.

Reply:
Thanks!  I didn't see that.

Reply:
done